src/test/java/org/folio/rest/api/InventoryHierarchyViewTest.java

```diff
@@ -95,6 +95,26 @@ public void setUp() {
     createItem(thirdFloorLocationId, "item barcode 2", "item effective call number 2", bookMaterialTypeId);
   }
 
+  @Test
+  public void ServerErrorWrittenOutOnDatabaseError() throws InterruptedException, ExecutionException, TimeoutException {
+
+    setFaultyStatisticalCode();    
+
+    params.put(QUERY_PARAM_NAME_SKIP_SUPPRESSED_FROM_DISCOVERY_RECORDS, "false");
+
+    List<JsonObject> instances = requestInventoryHierarchyViewUpdatedInstanceIds(params, response -> assertThat(response.getStatusCode(), is(200)));
+    UUID[] instanceIds = instances.stream()
+      .map(json -> UUID.fromString(json.getString("instanceId")))
+      .toArray(UUID[]::new);
+    
+    List<JsonObject> instancesData = requestInventoryHierarchyItemsAndHoldingsViewInstance(instanceIds, false, response -> assertThat(response.getStatusCode(), is(500)));
```

Comment:
Can the assertion be pushed down into the method in order to shorten this line?

Is there anywhere that calls `requestInventoryHierarchyViewUpdatedInstanceIds` using a different matcher that checks something other than the status code?

Reply:
> Can the assertion be pushed down into the method in order to shorten this line?

If I write my own version of the requestInventoryHierarchyViewUpdatedInstanceIds method, yes.  The way this one is written, you have to pass the assertion in as an argument.  I'll do so.

> Is there anywhere that calls `requestInventoryHierarchyViewUpdatedInstanceIds` using a different matcher that checks something other than the status code?

No, and I went back and forth a bit on wether I should add this-my concern here is that 500 errors are so generic-I'd like to be sure it's throwing one for the reason I expect.  I can remove if you think it's not necessary.

Reply:
> If I write my own version of the requestInventoryHierarchyViewUpdatedInstanceIds method, yes. The way this one is written, you have to pass the assertion in as an argument. I'll do so.

Given this is the only use, I don't think there is a need to create another intermediary.

Reply:
Please could the statement be split so that it isn't more than 100 characters per line.